### DISCUSSION TITLE
How to pass parameters to Chrome target?

### DISCUSSION THREAD

**Discussion by XFox111** [aklinker1/vite-plugin-web-extension#144] [Q&A] [answered]
I am checking out development containers in VS Code for developing web extensions. Unfortunately, Chrome in containers runs only with `--no-sandbox` flag enabled. How do I pass this flag into `vite dev`? It should be a chrome specific as I also sometimes use Firefox for debugging.

I currently have vanilla plugin setup with a few customizations.

**Comment by aklinker1** [comment, marked as answer]
See https://github.com/aklinker1/vite-plugin-web-extension/issues/22#issuecomment-1083299726 for how to pass CLI flags.
